src/MSALWrapper.Test/AuthFlowTest/AuthFlowBrokerTest.cs

```diff
@@ -396,6 +398,27 @@ public async Task BrokerAuthFlow_GetTokenInteractive_OperationCanceledException_
             authFlowBroker.ErrorsList[2].Message.Should().Be("Interactive Auth (with extra claims) timed out after 15 minutes.");
         }
 
+        /// <summary>
+        /// Ensure <see cref="IPCAWrapper.WithPromptHint"/> be invoked in <see cref="TokenFetcherPublicClient.GetTokenNormalFlowAsync"/>.
+        /// </summary>
+        /// <returns>The <see cref="Task"/>.</returns>
+        [Test]
+        public async Task GetTokenNormalFlowAsync_GetTokenInteractive_WithPromptHint()

```

Comment:
Is this test method name still appropriate? The method under test here is no longer called `GetTokenNormalFlowAsync`. I feel like we can probably arrive at a more accurate, terser name here.

Reply:
Yep, renamed it to be consistent with other tests in AuthFlowBroker.